docs/usage/shoot_pod_autoscaling_best_practices.md

```diff
@@ -0,0 +1,327 @@
+- If you have defined absolute limits (unrelated to the requests), configure VPA to only scale the requests or else it will proportionally scale the limits as well, which can easily become useless (way beyond the threshold of unhealthy behavior) or absurd (larger than node capacity):
+  ```
+  spec:
+    resourcePolicy:
+      containerPolicies:
+      - controlledValues: RequestsOnly
+        ...
+  ```
+  If you have defined relative limits (related to the requests), the default policy to scale the limits proportionally with the requests is fine, but the gap between requests and limits must be zero for QoS `Guaranteed` and should best be small for QoS `Burstable` to avoid useless or absurd limits either, e.g. prefer limits being 5 to at most 20% larger than requests as opposed to being 100% larger or more.
+- As a rule of thumb, set `minAllowed` to the highest observed VPA recommendation (usually during the initialization phase or during any periodical activity) for an otherwise practically idle container, so that you avoid needless trashing (e.g. resource usage calms down over time and recommendations drop consecutively until eviction, which will then lead again to initialization or later periodical activity and higher recommendations and new evictions).<br/>⚠️ You may want to provide higher `minAllowed` values, if you observe that up-scaling takes too long for CPU or memory for a too large percentile of your workload. This will get you out of the danger zone of too few resources for too many pods at the expense of providing too many resources for a few pods. Memory may react faster than CPU, because CPU throttling is not visible and memory gets aided by OOM bump-up incidents, but still, if you observe that up-scaling takes too long, you may want to increase `minAllowed` accordingly.
+- As a rule of thumb, set `maxAllowed` to your theoretical maximum load, flanked with alerts to detect erroneous run-away usage or the actual nearing of your practical maximum load, so that you can intervene. However, VPA can easily recommend requests larger than what is allocatable on a node, so you must either ensure large enough nodes (Gardener can scale up from zero, in case you like to define a low-priority worker pool with more resources for very large pods) and/or cap VPA's target recommendations using `maxAllowed` at the node allocatable remainder (after daemonset pods) of the largest eligible machine type (may result in under-provisioning resources for a pod). Use your monitoring and check maximum pod usage to decide about the maximum machine type.
+
+## Recommendations in a Box
```

Comment:
This section is what I was looking for ❤️ 

I think it should be linked from https://github.com/gardener/gardener/blob/master/docs/development/component-checklist.md#scalability

Reply:
Ah, thank you! Let me add it then!

Reply:
Done, but I had to rewrite the corresponding sub section, so it needs a review now too, I believe.